timesketch/ui/graphs/main.component.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
Angular2+ components do not need single template root components, so you can remove this div (if it doesn't serve any other purpose).

Reply:
It is a flexbox container with fixed height, so it has a purpose. But this is good to know.

Reply:
Then I wonder why not make the component element itself the flexbox container?

Reply:
Done.